modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/QueryMemoryManager.java

```diff
@@ -333,6 +333,14 @@ public long freeMemory() {
         return memoryLimit() - reserved();
     }
 
+    /**
+     * @return Percent of currently available memory for SQL queries on the node.
```

Comment:
What reason has the metric?  If some value could be evaluated by other values it must be done outside of the product.